client/src/themes/theme.js

```diff
@@ -1,11 +1,20 @@
 import { createMuiTheme } from "@material-ui/core";
 
 export const theme = createMuiTheme({
+  overrides: {
+    MuiChip: {
+      root: {
+        borderRadius: 0,
+        margin: "5px",
+      },
+    },
+  },
   typography: {
-    fontFamily: '"Roboto"'
+    fontFamily: '"Montserrat"',
```

Comment:
Can you change this back, the previous font matches the mocks better

Reply:
I'll change this back